gradle/versions.gradle

```diff
@@ -8,7 +8,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.0.2'
```

Comment:
Besides is `android_x_annotation` used anywhere?

Reply:
yep